### PR TITLE
Drupal 9 compatibility

### PR DESCRIPTION
**Pull request by nkoporec**

### CODE REVIEW

File: .ddev/commands/web/install [hunk 1]
```diff
@@ -30,12 +30,6 @@ drush queue-run activity_logger_message
 drush queue-run activity_creator_logger
 drush queue-run activity_creator_activities
 drush queue-run activity_send_email_worker
-echo "trigger a search api re-index"
-drush php-eval 'drush_search_api_reset_tracker();';
```
Comment (navneet0693) [inline .ddev/commands/web/install:34]: This can be replaced by `drush ev 'node_access_rebuild()';`

File: .ddev/commands/web/install [hunk 2]
```diff
@@ -30,12 +30,6 @@ drush queue-run activity_logger_message
...
 
 echo "rebuild node access"
 drush php-eval 'node_access_rebuild()';
-
-# development modules.
-echo "enabling devel modules"
-drush en social_devel -y
```
Comment (navneet0693) [inline .ddev/commands/web/install:41]:
```
echo "enabling devel modules"
drush en -y config_update config_update_ui devel devel_generate dblog views_ui field_ui contextual
```

File: .ddev/config.yaml [hunk 3]
```diff
@@ -10,19 +10,12 @@ additional_hostnames: []
 additional_fqdns: []
 mariadb_version: "10.3"
```
Comment (navneet0693) [inline .ddev/config.yaml]: Maybe bump it up to 10.7

Comment (nkoporec) [reply .ddev/config.yaml]: Unfortunately ddev supports only 10.5, so setting this for now.

**Pull request merged**